screens/pilot/PilotProfileSetupPageTwoScreen.js

```diff
@@ -272,6 +272,24 @@ const styles = StyleSheet.create({
   pickerButtonWrapper: {
     alignItems: "center",
   },
+  backButton: {
+    marginTop: 20,
+    marginBottom: 40,
+    width: 60,
+    height: 30,
+    backgroundColor: "#092455",
+    borderRadius: 30,
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  backButtonText: {
+    color: "white",
+    textAlign: "center",
+    // marginBottom: 40,
```

Comment:
rm

Reply:
rm??? Shiz, not sure what that means, lol